tests/tdastro/sources/test_physical_models.py

```diff
@@ -1,10 +1,31 @@
+import pytest
+from astropy.cosmology import WMAP9
 from tdastro.sources.physical_model import PhysicalModel
 
 
 def test_physical_model():
     """Test that we can create a PhysicalModel."""
     # Everything is specified.
-    model1 = PhysicalModel(ra=1.0, dec=2.0, distance=3.0)
+    model1 = PhysicalModel(ra=1.0, dec=2.0, distance=3.0, redshift=0.0)
     assert model1.ra == 1.0
     assert model1.dec == 2.0
     assert model1.distance == 3.0
+    assert model1.redshift == 0.0
+
+    # Derive the distance from the redshift using the example from:
+    # https://docs.astropy.org/en/stable/api/astropy.cosmology.units.redshift_distance.html
+    model2 = PhysicalModel(ra=1.0, dec=2.0, redshift=1100.0, cosmology=WMAP9)
```

Comment:
Let's use Planck18 instead

Reply:
I used it here, but kept the `WMAP9` test in `test_cosmology.py` since I have the answer from the astropy example and can check against that.